consensus/polybft/consensus_metrics.go

```diff
@@ -32,6 +32,9 @@ func updateBlockMetrics(currentBlock *types.Block, parentHeader *types.Header) e
 
 	// number of rounds needed to seal a block
 	metrics.SetGauge([]string{consensusMetricsPrefix, "rounds"}, float32(extra.Checkpoint.BlockRound))
+	metrics.SetGauge([]string{consensusMetricsPrefix, "block_transactions"}, float32(len(currentBlock.Transactions)))
```

Comment:
Isn't this one the same as `num_txs`?
https://github.com/0xPolygon/polygon-edge/pull/1257/files#diff-a8c88bd4c811653ef247e206d461aafa89a62a0b89d08425052924a13c7d879dR26

Reply:
I agree with Stefan, I would just change
`metrics.SetGauge([]string{consensusMetricsPrefix, "num_txs"}, float32(len(currentBlock.Body().Transactions)))`
to
`metrics.SetGauge([]string{consensusMetricsPrefix, "num_txs"}, float32(len(currentBlock.Transactions)))`

Reply:
@b00ris either remove this and change that metric according to Igor's comment , or `num_txs` and keep this one (although in that case, it is necessary to update Polygon Edge docs: https://github.com/maticnetwork/matic-docs/blob/master/docs/edge/configuration/prometheus-metrics.md).
Though I agree that `block_transactions` is better naming than `num_txs`.

```suggestion
```